### PR TITLE
[account] usability improvements

### PR DESCRIPTION
**Pull request by rga-odoo**
Task: https://www.odoo.com/web#id=35329&view_type=form&model=project.task&action=327&menu_id=
pad: https://pad.odoo.com/p/r.b4cb5dca1c9b83b5c5ca53211368bf7a

### CODE REVIEW

**Comment by msh-odoo** [comment]
LGTM

File: addons/account/static/src/xml/account_reconciliation.xml [hunk 1]
```diff
@@ -24,11 +24,15 @@
             <button class="btn btn-default o_automatic_reconciliation pull-right" title="Let odoo try to reconcile entries for the user">Automatic reconciliation</button>
             <div class="notification_area"/>
         </div>
+        <div class="text-muted">
+            <div><strong> Tip:</strong></div>
+            <div>(1)  Hit CTRL-Enter to reconcile all the balanced items in the sheet.</div>
```
Comment (qdp-odoo) [inline addons/account/static/src/xml/account_reconciliation.xml]: extra blank

**Comment by rdeodoo** [comment]
@qsm-odoo Task cancelled a few months ago, can we close?

**Comment by qdp-odoo** [comment]
@rdeodoo cleaning up github? And who's that @qsm-odoo ?

**Comment by rdeodoo** [comment]
Well yeah, it hurts everytime I see 1500+ PR.
800 from RD, including 200 already fixed from somewhere else, 200 from cancelled task
What's crazy is there is 'ghost' RD task without any task id, fixing a real bug, without any reviewer, that's a waste..
I would sleep better if we can go back bellow 1000 :pray: 

It would be easier if anyone could check and clean his open PR pipe honestly.
Last time I cleaned issues/PR last year, @mart-e sent an email to everyone a few days after, that helped :yum: 

@qsm-odoo is our new ping pong champion, all hail to the king. He fu**ing stole my :crown:

**Comment by qdp-odoo** [comment]
Until Monday only... :smiling_imp:

**Comment by rdeodoo** [comment]
Don't tell anyone but I think @sbuhl is the real champion. Il a roulé sur @qsm-odoo (Yeah my english is limited sorry :crying_cat_face:)

**Pull request closed without merging**